sunpy/map/tests/test_mapbase.py

```diff
@@ -1494,3 +1495,9 @@ def test_map_arithmetic_operations_raise_exceptions(aia171_test_map, value, warn
         # the map test
         with warn_context:
             _ = value / aia171_test_map
+
+
+def test_parse_fits_units():
+    # Check that we parse a BUNIT of G correctly.
+    out_unit = GenericMap._parse_fits_unit("G")
```

Comment:
```suggestion
    out_unit = GenericMap._parse_fits_unit("Gauss")
```

Reply:
The BUNIT keyword should be G?

Reply:
This test as written wasn't testing the change you made to the code. This would make it actually test the change you made, I have no idea what you are trying to achieve because there is no description in the PR.

Reply:
Its ok, just approve it and go to sleep.